snakemd/elements.py

```diff
@@ -354,24 +340,27 @@ class Paragraph(Block):
     A paragraph is a standalone block of text. Paragraphs can be
     formatted in a variety of ways including as blockquotes.
 
-    :param Iterable[Inline | str] content: a "list" of text objects to render as a paragraph        
+    :param str | Iterable[Inline | str] content: a "list" of text objects to render as a paragraph        
     :param bool quote: the quote state of the paragraph;
         set True to convert the paragraph to a blockquote (i.e., True -> > quote)
     """
 
-    def __init__(self, content: Iterable[Inline | str], quote: bool = False):
+    def __init__(self, content: str | Iterable[Inline | str], quote: bool = False):
         super().__init__()
         self._content: list[Inline] = self._process_content(content)
         self._quote = quote
 
     @staticmethod
-    def _process_content(content) -> None:
-        processed = []
-        for item in content:
-            if isinstance(item, str):
-                processed.append(Inline(item))
-            else:
-                processed.append(item)
+    def _process_content(content) -> list[Inline]:
```

Comment:
Again, using `list` or any container data type is python 3.10+. Instead, you'll need to use `List`

Reply:
Nice catch! I think there's a way around this with the future import. Seems to work now after I updated the action to properly import the markdown module for testing.